components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,84 @@
+const reddit = require("../../reddit.app.js");
+const get = require("lodash.get");
+
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
+  description: "Emmits a new link submitted to a subreddit",
+  version: "0.0.2",
+  props: {
+    reddit,
+    timer: {
```

Comment:
Just a style comment, but I like to put the timer schedule _below_ the app-specific props, since the app-specific props are normally more relevant for the user. So I'd just swap the order of the `timer` and `subreddit` props.